dashboard/dashboard-intro.md

```diff
@@ -0,0 +1,63 @@
+---
+title: TiDB Dashboard Introduction
+summary: Introduce TiDB Dashboard.
+category: how-to
+---
+
+# TiDB Dashboard Introduction
+
+TiDB Dashboard is a graphical interface provided by TiDB since v4.0, which is used to monitor and diagnose TiDB clusters. TiDB Dashboard is built into the PD component of TiDB and does not require independent deployment.
```

Comment:
```suggestion
TiDB Dashboard is a Web UI for monitoring, diagnosing and managing the TiDB cluster, available since v4.0. It is built into the PD component and does not require an independent deployment.
```